api/swagger.yaml

```diff
@@ -5436,6 +5436,35 @@ definitions:
         example:
           - "/etc/cdi"
           - "/var/run/cdi"
+      Containerd:
+        $ref: "#/definitions/ContainerdInfo"
+        x-nullable: true
+
+  ContainerdInfo:
+    description: |
+      Information for connecting to the containerd instance that is used by the daemon.
+      This is included for debugging purposes only.
+      Tampering with the containerd instance may cause unexpected behavior.
+    type: "object"
+    properties:
+      Address:
+        description: "The address of the containerd socket."
+        type: "string"
+      Namespaces:
+        description: "The namespaces that are used by containerd."
+        type: "object"
+        properties:
+          Containers:
+            description: "The namespace that is used for containers."
+            type: "string"
+          Plugins:
+            description: "The namespace that is used for plugins."
+            type: "string"
+    example:
+      Address: "/run/containerd/containerd.sock"
+      Namespaces:
+        Containers: "moby"
+        Plugins: "plugins.moby"
```

Comment:
I'm a bit on the fence on exposing this in the `/info` endpoint. I think we need to consider if there's a better way to expose the containerd API, possibly in combination with https://github.com/moby/moby/issues/44369.

Even though it's clearly stated that it's for debugging purposes only, there's [Hyrum's Law](https://www.hyrumslaw.com/)...

Reply:
Not going to block, if other maintainers think it's okay though.

Reply:
The API is no more exposed than it was before, though.